### ISSUE TITLE
[Bug]: file path with spaces in not properly decoded during tests

### ISSUE THREAD

**Issue by simpletasks** [testcontainers/testcontainers-java#9025]
### Module

Core

### Testcontainers version

master

### Using the latest Testcontainers version?

Yes

### Host OS

Windows

### Host Arch

x86

### Docker version

```shell
26.1.1
```

### What happened?

On local build of the testcontainer project with gradlew clean build -x test,
core module tests failed because spaces in the path of the jar file.

### Relevant log output

```shell
Caused by: java.lang.RuntimeException: java.nio.file.NoSuchFileException: C:\Users\Krunoslav%20Magazin\projects\testcontainers-java\core\build\libs\testcontainers.jar
	at org.testcontainers.AbstractJarFileTest.<clinit>(AbstractJarFileTest.java:21)
```

### Additional Information

Create a directory with space in the name. Inside the directory put testcontainers-java project and try to build it.

**Comment by simpletasks** [comment]
#9026 merge request

**Comment by eddumelendez** [comment]
Fixed by #9026
